opentelemetry-cpp.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: opentelemetry-cpp
+  version: 1.17.0
+  epoch: 0
+  description: The OpenTelemetry C++ Client
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - abseil-cpp-dev
+      - benchmark-dev
+      - busybox
+      - c-ares-dev
+      - cmake
+      - gcc
+      - glibc-dev
+      - grpc-dev
+      - gtest-dev
+      - icu-dev
+      - make
+      - openssl-dev
+      - pkgconf
+      - protobuf-dev
+      - re2-dev
+      - systemd-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/open-telemetry/opentelemetry-cpp
+      tag: v${{package.version}}
+      expected-commit: fa0a5200ddbe9ae6f487a228e658aedc102dab56
+
+  - runs: |
+      mkdir build
+      cd build
+      cmake -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DBUILD_SHARED_LIBS=ON -DWITH_OTLP_GRPC=ON -DWITH_ABSEIL=ON ..
```

Comment:
We may be able to leverage our cmake pipelines here

Reply:
Done.